### DISCUSSION TITLE
Welcome to open-cravat Discussions!

### DISCUSSION THREAD

**Discussion by rkimoakbioinformatics** [KarchinLab/open-cravat#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kpagel** [comment]
Hello all!! Let's do a round of introductions. 

I'm Kym Pagel from Karchin Lab at Johns Hopkins. My contributions to this project help shape development and span from finding annotation sources to hands-on work with collaborators.

**Comment by kmoad** [comment]
Hi!

I'm Kyle from In Silico Solutions, which is partnered with Karchin Lab to developing OpenCRAVAT. I'm one of the developers of the base oc code, as well as the developer of many of the annotators at [open-cravat-modules-karchinlab](https://github.com/KarchinLab/open-cravat-modules-karchinlab).

**Comment by rkimoakbioinformatics** [comment]
Hi, I am Rick Kim from In Silico Solutions. I am a lead architect of OpenCRAVAT.

**Comment by RachelKarchin** [comment]
I'm Rachel Karchin.  I work to shape and sustain OpenCRAVAT.  I also use
OpenCRAVAT in my research!

- RK
